torchmetrics/average.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+import torch
+
+from .metric import Metric
+
+class AverageMeter(Metric):
+    """Computes the average of a stream of values.
+
```

Comment:
probably need a bit of formatting
```suggestion

Args:
            compute_on_step:
            Forward only calls ``update()`` and return ``None`` if this is set to ``False``.
        dist_sync_on_step:
            Synchronize metric state across processes at each ``forward()``
            before returning the value at the step
        process_group:
            Specify the process group on which synchronization is called.
            default: ``None`` (which selects the entire world)
        dist_sync_fn:
            Callback that performs the allgather operation on the metric state. When ``None``, DDP
            will be used to perform the allgather

```